cmd/botkube/main.go

```diff
@@ -128,6 +128,9 @@ func run() error {
 	// Set up the filter engine
```

Comment:
Please also cover the new functionality with some tests. I think some unit tests will be enough. It would be also awesome to test it as a part of E2E tests, but Botkube restart could be a bit problematic 🤔

Reply:
In E2E tests, can we at least list actions (`list actions`)? That will be easy to test as the output is predictable 🤔
As I wrote, enabling/disabling could be a bit harder (we'd need to wait for restart, and that will result in Botkube "welcome" message on both channels), so for me enable/disable are nice to have. The only thing we need to consider is that everything we won't cover in automated tests we'll need to test manually.